cluster/src/main/java/com/hartwig/pipeline/sbpapi/SbpRunResult.java

```diff
@@ -0,0 +1,37 @@
+package com.hartwig.pipeline.sbpapi;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.hartwig.pipeline.execution.PipelineStatus;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableSbpRunResult.class)
+@JsonSerialize(as = ImmutableSbpRunResult.class)
+public interface SbpRunResult {
```

Comment:
While it might be inconsistent, this would also be an opportunity to not add more SBP legacy.